examples/elixir/stm32/MultiBlink.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MultiBlink do
+  def start do
+    spawn(__MODULE__, :blink, [{:d, 12}, 1000])
+    spawn(__MODULE__, :blink, [{:d, 13}, 500])
+    spawn(__MODULE__, :blink, [{:d, 14}, 1500])
+    spawn(__MODULE__, :blink, [{:d, 15}, 300])
+
+    loop()
+  end
+
+  def loop do
+    loop()
+  end
+
+  def blink(gpio, interval_ms) do
+    gpio_driver = GPIO.open();
```

Comment:
No need for trailing semicolon

Reply:
Oops, that was already there in the old file, nice catch